spec/project_spec.rb

```diff
@@ -27,6 +27,16 @@
       end
     end
 
+    it 'require a nicely formatted `VersionAdded` metadata for all cops' do
+      cop_names.each do |name|
+        version = config[name]['VersionAdded']
+        expect(version.nil?).to(be(false),
+                                "VersionAdded is required for #{name}.")
+        expect(version).to(match(/\A\d+\.\d+\z/),
+                           "#{version} is an illegal format for #{name}.")
```

Comment:
Should we include an indication of what the correct format looks like?

Reply:
I updated this PR. The following is an example. What do you think about this?

```console
% bundle exec rspec spec/project_spec.rb:31
(snip)

Failures:

  1) RuboCop Project default configuration file require a nicely formatted `VersionAdded` metadata for all cops
     Failure/Error:
       expect(version).to(match(/\A\d+\.\d+\z/),
                          "#{version} should be format ('X.Y') for #{name}.")

       0.13.1 should be format ('X.Y') for Style/LambdaCall.
       Diff:
       @@ -1,2 +1,2 @@
       -/\A\d+\.\d+\z/
       +"0.13.1"

     # ./spec/project_spec.rb:35:in `block (4 levels) in <top (required)>'
     # ./spec/project_spec.rb:31:in `each'
     # ./spec/project_spec.rb:31:in `block (3 levels) in <top (required)>'
```